ffs/scheduler/scheduler.go

```diff
@@ -204,8 +208,24 @@ func (s *Scheduler) GetLogs(ctx context.Context, c cid.Cid) ([]ffs.LogEntry, err
 	return lgs, nil
 }
 
+// Cancel cancels an executing Job.
+func (s *Scheduler) Cancel(jid ffs.JobID) error {
```

Comment:
The canceling feature is simple.
Each executing job has a dummy channel that should be closed if some wants to cancel it.